src/linker/Linker/Driver.cs

```diff
@@ -674,11 +674,20 @@ public bool Run (ILogger customLogger = null)
 
 			try {
 				p.Process (context);
+			} catch (Exception ex) {
+				Console.Error.WriteLine ("IL Linker exited with error:");
+				if (ex is LinkerFatalErrorException le) {
+					context.LogMessage (le.MessageContainer);
+					if (le.InnerException != null)
```

Comment:
This should not be necessary if `LinkerFatalErrorException::ToString` does not work please fix it there